### PR TITLE
WCP-1433 Fix linux compilation 2

### PR DESCRIPTION
**Pull request by rafalgliniak**

### CODE REVIEW

File: DynamicGui/CMakeLists.txt [hunk 1]
```diff
@@ -15,15 +15,17 @@ target_link_libraries(DynamicGui PUBLIC
                       Qt5::QmlWorkerScript
                       zmq)
 if(CMAKE_BUILD_TYPE MATCHES Release)
-    target_compile_definitions(DynamicGui PUBLIC ZMQ_STATIC)
+    if(WIN32)
+        target_compile_definitions(DynamicGui PUBLIC ZMQ_STATIC)
```
Comment (m-karcz) [inline DynamicGui/CMakeLists.txt:19]: but ZMQ should be static linked...

Comment (rafalgliniak) [reply DynamicGui/CMakeLists.txt:19]: why?

Comment (m-karcz) [reply DynamicGui/CMakeLists.txt:19]: simulator is deployed as single exe and zmq is required for dynamic ui tab

Comment (rafalgliniak) [reply DynamicGui/CMakeLists.txt:19]: we do deployment of shared libraries on linux, see deployment script in this review

**Review by van9konst — APPROVED** [review]

**Pull request merged**